packages/filebrowser/src/crumbs.ts

```diff
@@ -309,6 +319,15 @@ export class BreadCrumbs extends Widget {
   private _hasPreferred: boolean;
   private _crumbs: ReadonlyArray<HTMLElement>;
   private _crumbSeps: ReadonlyArray<HTMLElement>;
+  private _fullPath: boolean;
+
+  get fullPath(): boolean {
```

Comment:
Can you please move these two methods to above the private/protected methods (around the line 116) and add a documentation strings before the getter?